st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
```

Comment:
Hmm, we need to think this through. I am not convinced. I can see these being combined into formats after all that is the entire point of a format. I see being able to support regex in formats as useful.